client/components/additionalRepos/additional-repos.html

```diff
@@ -0,0 +1,22 @@
+<label>Additional repositories</label>
+<button class="btn btn-xs btn-success" type="button" ng-click="addRepository()">+</button>
+
+<div ng-repeat="repo in model track by $id($index)" ng-if="$index >= 1" ng-class="{ 'has-success': editProjectForm['repository_' + $index].$valid && submitted,
+  'has-error': editProjectForm['repository_' + $index].$invalid && submitted }">
+
+  <div class="form-group">
+     <div class="input-group col-md-12">
+       <input type="url" name="repository_{{$index}}" class="repo form-control" ng-model="model[$index]" required/>
+       <span class="input-group-btn">
+         <button class="btn btn-danger repoButton" class="form-control" type="button" ng-click="removeRepository($index)">-</button>
```

Comment:
Lets use one of these icons instead of the -?
http://fontawesome.io/icon/times/
http://fontawesome.io/icon/trash/
http://fontawesome.io/icon/minus/
http://fontawesome.io/icon/minus-square/